### PR TITLE
Document how to escape emoji

### PR DESCRIPTION
**Pull request by svoop**
Related to https://github.com/gollum/gollum-lib/pull/275

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Gollum comes with the following command line options:
 | --page-file-dir   | [PATH]    | Specify the subdirectory for all pages. If set, Gollum will only serve pages from this directory and its subdirectories. Default: repository root. |
 | --css             | none      | Tell Gollum to inject custom CSS into each page. Uses `custom.css` from repository root.<sup>3,5</sup> |
 | --js              | none      | Tell Gollum to inject custom JS into each page. Uses `custom.js` from repository root.<sup>3,5</sup> |
-| --emoji           | none      | Parse and interpret emoji tags (e.g. :heart:). |
+| --emoji           | none      | Parse and interpret emoji tags (e.g. `:heart:`) unless when the leading colon is backslashed (e.g. `\:heart:`). |
```
Comment (dometto) [inline README.md]: Perhaps `except when`?

Comment (svoop) [inline README.md]: Absolutely!

**Comment by dometto** [comment]
:+1: Could you add a sentence on this in the [5.0 release notes](https://github.com/gollum/gollum/wiki/5.0-release-notes)? Thanks!

**Comment by svoop** [comment]
> Could you add a sentence on this in the 5.0 release notes? Thanks!

Sure, I've added an example. (Yeah, frontmatter in 5.0, cool!!)

**Comment by svoop** [comment]
Thanks for merging!

**Pull request merged**